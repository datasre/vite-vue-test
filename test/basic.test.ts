/*
 * @Description  : 
 * @Version      : 1.0
 * @Author       : he_jian
 * @Date         : 2025-07-09 15:38:09
 * @LastEditors  : he_jian
 * @LastEditTime : 2025-07-09 15:40:24
 * @FilePath     : \\vite-vue-test\\test\\basic.test.ts
 * Copyright (c) 2025 by he_jian, All Rights Reserved. 
 */
import {assert, describe, it, expect} from 'vitest'

describe('basic', () => {
    it('foo', () => {
        assert.equal(Math.sqrt(4), 2)
    })
    it('bar', () => {
        expect(1+1).eq(2)
    })
    it('baz', () => {
        expect({foo:'bar'}).toMatchSnapshot();
    })
})