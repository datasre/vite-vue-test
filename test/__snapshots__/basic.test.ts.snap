// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`basic > baz 1`] = `
{
  "foo": "bar",
}
`;
